keyloader/keyloader.go

```diff
@@ -218,7 +218,65 @@ func configFactory() interface{} {
 ** CONSTRUCTORS
  */
 
+// NewKey returns a symmecrypt.Key object configured from a number of KeyConfig objects.
+// If several KeyConfigs are supplied, the returned Key will be composite.
+// A composite key encrypts with the latest Key (based on timestamp) and decrypts with any of they keys.
+// The KeyConfig parameters are expected to be sorted, with the latest key as first element.
```

Comment:
why not `sort.Slice(cfgs, ....)` to sort systemically by timestamp ?

Reply:
Yeah, that's fair. I didnt do it because LoadKey already does it (via configstore), so it seemed acceptable for the lower-level func to have a documented constraint instead of just doing it.
Will think it over, probably will sort.

Reply:
Added the sort